### PR TITLE
Adds Podmorph humans and pod-printer;

### PR DESCRIPTION
**Pull request by Andromeda-K22**
New:
Podmorph Humans;
https://cdn.discordapp.com/attachments/426599198083776522/431613742283751434/unknown.png
"With cloning on the forefront of human scientific advancement, cheap mass production \
of bodies is a very real and rather ethically grey industry. Podmorphs are designed for those who can't accept vatgrowns. Horrifying amalgams of flesh and machine, these artificial humans exist as a cheap substitute for actual vatgrown bodies. Their bodies are frail, and their flesh weak. These amalgams are often used on the Rim, where cloning is often unavailable and lace technology common."

Podmorph humans take:
10% less toxloss
25% more bruteloss
and 40% more burnloss. 

Podmorphs have all synthetic organs, making them vulnerable to EMPs.

Pod-Printer:
https://cdn.discordapp.com/attachments/413650928458006528/431644142284570626/replicatoron.gif

The pod-printer can hold 55,000cm2 of metal and biomass. A pod-person takes 15,000cm2 of biomass and metal, plus twenty minutes, to print. If the printer looses power during the print, the pod-person inside is cancelled and the resources put back into storage.

### CODE REVIEW

**Comment by Andromeda-K22** [comment]
Ignore the wonky hand. I fixed that.

File: code/modules/podprinter/printer.dm [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+	var/printing_name = null
+	var/can_print = 1 //Can we print?
+	var/printing = 0 //Running.
+	var/metal = 0
+	var/biomass = 0
+	var/max_metal = 0 //How much metal can we hold
+	var/max_biomass = 0 //How much biomass can we hold
+	var/list/choices = list("Yes", "No")
+	var/list/gender_choices = list("male", "female")
+	var/list/running_sounds = list("hisses and hums.", "whirrs and beeps", "clicks and thumps", "thrums and vibrates")
```
Comment (orelbon) [inline code/modules/podprinter/printer.dm]: Stray period.

**Comment by orelbon** [comment]
Will review more tomorrow, its late for me now.

File: code/modules/podprinter/printer.dm (hunk 1 above)
Comment (MalfunctioningAI) [reply code/modules/podprinter/printer.dm]: Where?

**Comment by MalfunctioningAI** [comment]
Will my PR and this one conflict, considering both touch the subspecies file?

**Comment by Andromeda-K22** [comment]
I did say that i was working on pod people a while ago, but in all likelyhood this may create merge conflicts.   Without changing the focus of this PR, the only way i can see this working is if you close your PR and redo your changes after this gets merged.

**Comment by MalfunctioningAI** [comment]
Or I could add your part of the subspecies file into my own PR.

**Comment by MalfunctioningAI** [comment]
I'll wait until this is merged, then copypaste your changes into my own PR.

File: code/modules/podprinter/printer.dm (hunk 1 above)
Comment (MalfunctioningAI) [reply code/modules/podprinter/printer.dm]: "hisses and hums."

**Pull request merged**